flake.nix

```diff
@@ -44,7 +44,7 @@
             buildInputs = [ pkgs.bashInteractive ];
             shellHook = ''
               if [ ! -f .git/hooks/pre-commit ]; then
-                pre-commit install
+                echo "You can run 'pre-commit install' to install git commit hooks if you want them."
               fi
 
               pnpm install
```

Comment:
Since the pre-commit install has been removed, should we remove the pnpm install too? It's not exactly fast when you consider how fast a shell typically starts.

Reply:
I unfortunately lack some understanding about how nix works, but I would imagine that when creating the environment, we want to do a pnpm install, but not when just activating the shell. Not sure if that makes sense / is possible